src/content/en/fundamentals/native-hardware/recording-audio/index.md

```diff
@@ -0,0 +1,235 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Most browsers can get access to the user's microphone.
+
+{# wf_updated_on: 2016-08-22 #}
+{# wf_published_on: 2016-08-23 #}
+
+# Recording Audio from the User {: .page-title }
+
+{% include "web/_shared/contributors/paulkinlan.html" %}
+
+Many browsers now have the ability to access video and audio input from the user. However,
+depending on the browser it might be a full dynamic and inline experience, or it could 
+be delegated out to another app on the user's device.
+
+## Start simple and progressively
+
+The easiest thing that we can do is simply ask the user for a pre-recorded file. This can be 
+done by creating a simple file input element and adding an `accept` filter that indicates
+we can only accept audio files and ideally we will get them directly from the microphone.
+
+```
+<input type="file" accept="audio/*;capture=microphone">
+```
+
+This method will work on all platforms. On desktop it will prompt the user to upload a file from
+the file system (ignoring `capture=microphone`). In Safari on iOS it will open up the microphone 
+app, allowing you to record audio and then send it back to the web page; on Android it will 
+give the user of which app to use record the audio in before sending it back to the web page.
+
+Once the user has finished recording and they are back in the web site. You need to somehow get
+ahold of the file data. You can get quick access to this data by attaching an `onchange` event 
+to the input element and then reading the `files` property of the event object.
+
+<pre class="prettyprint">
```

Comment:
Same here, you don't have to wrap JavaScript in `<pre class="prettyprint">` you can simply put the whole block in and indent it 4 spaces.

It also means you don't need to escape HTML entities like <